### PR TITLE
Add ButlerRuleFailedBoard to NotificationType

### PR DESCRIPTION
**Pull request by techfooninja**
I know this repo isn't maintained anymore, but wanted to include this PR in case anyone else ran into the same bug I did. Thanks for all you've done!

### CODE REVIEW

**Comment by gregsdennis** [comment]
Thanks for the PR. I can pull this in. Would you also mind bumping the minor version?

There's also an issue for another event type that's missing.  If you could add that, is appreciate it.

Those two things, then I can merge.

**Comment by techfooninja** [comment]
Do you know what the other event type is?

**Comment by gregsdennis** [comment]
https://github.com/gregsdennis/Manatee.Trello/issues/342

**Comment by techfooninja** [comment]
Awesome, thanks. I just pushed the changes.

File: Manatee.Trello/Notification.cs [hunk 1]
```diff
@@ -146,6 +146,7 @@ static Notification()
 					{NotificationType.RemovedFromOrganization, n => $"{n.Creator} removed member {n.Data.Member} from organization {n.Data.Organization}."},
 					{NotificationType.MentionedOnCard, n => $"{n.Creator} mentioned you on card {n.Data.Card}: '{n.Data.Text}'."},
 					{NotificationType.UpdateCheckItemStateOnCard, n => $"{n.Creator} updated checkItem {n.Data.CheckItem} on card {n.Data.Card}."},
+					{NotificationType.CreateCheckItem, n => $"{n.Creator} added checkItem {n.Data.CheckItem} on card {n.Data.Card}" },
```
Comment (gregsdennis) [inline Manatee.Trello/Notification.cs:149]: I think this was actually an action (`ActionType`).

File: Manatee.Trello/Manatee.Trello.csproj [hunk 2]
```diff
@@ -15,7 +15,7 @@
     <SignAssembly>True</SignAssembly>
     <AssemblyOriginatorKeyFile>Manatee.Trello.snk</AssemblyOriginatorKeyFile>
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
-    <FileVersion>4.3.0.0</FileVersion>
+    <FileVersion>4.3.1.0</FileVersion>
```
Comment (gregsdennis) [inline Manatee.Trello/Manatee.Trello.csproj:18]: Should be 4.4.0 because you're adding a new member.

File: Manatee.Trello/Manatee.Trello.csproj [hunk 3]
```diff
@@ -5,7 +5,7 @@
     <TargetFrameworkIdentifier>.NETStandard</TargetFrameworkIdentifier>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <Description>A fully object-oriented .Net wrapper for the TrelloAPI.</Description>
-    <Version>4.3.0</Version>
+    <Version>4.3.1</Version>
```
Comment (gregsdennis) [inline Manatee.Trello/Manatee.Trello.csproj:8]: 4.4.0

**Review by gregsdennis — CHANGES_REQUESTED** [review]
A couple things. Thanks.

**Comment by techfooninja** [comment]
Resolved your comments. Thanks for the feedback.

**Review by gregsdennis — APPROVED** [review]
Thanks.  I'll figure out what's happening with the build.

**Comment by techfooninja** [comment]
I think it may be configuration with the account. I think Trello moved to Atlassian auth a while back.

**Comment by gregsdennis** [comment]
The oauth1 auth still works with the api.  I can run the tests locally.  I'll just update the build. 🤞

**Pull request merged**